__tests__/games.test.js

```diff
@@ -0,0 +1,45 @@
+const request = require("supertest")
+const app = require("../app.js")
+const db = require("../db/connection.js")
+const seed = require("../db/seeds/seed")
+const testData = require("../db/data/test-data/index")
+
+beforeEach(() => seed(testData))
+
+afterAll(() => {
+    db.end();
+  });
+
+  describe('GET /api/categories', () => {
+      test('200: responds with an array of objects each with a slug and description key ', () => {
+          return request(app)
+          .get("/api/categories")
+          .expect(200)
+          .then(({body}) => {
+
+            const categories = body;
```

Comment:
categories should be a key on the body, not the actual body itself. It should be structured as such
body: {
    categories : [.....]
}